packages/analytics/src/events.ts

```diff
@@ -152,6 +152,7 @@ type UserSettingValue =
   | { bitcoin_unit: 'bitcoin' | 'satoshi' }
   | { email_address: string }
   | { fiat_currency: string }
+  | { quote_currency: string }
```

Comment:
@camerow what should we do with this event? I added a new `event` as I'd like to rename it as `quote_currency` but I didn't want to remove `fiat_currency` either

Reply:
I'll keep both for now and we can retire `fiat_currency` / merge them in future